op-guide/configuration.md

```diff
@@ -94,6 +94,16 @@ The default TiDB ports are 4000 for client requests and 10080 for status report.
 + default: true
 + The value can be (true) or (false). (true) is to enable join without any equal conditions. (false) is to disable it.
 
+### --proxy-protocol-networks
++ PROXY Protocol allowed proxy server IP addresses or CIDRs (split by `,`).
```

Comment:
This sentence is ambiguous, does this mean:
The allowed proxy server IP addresses or CIDRs (split by `,`) that is allowed by the PROXY Protocol?